cdc/sink/codec/maxwell.go

```diff
@@ -248,20 +297,10 @@ func (d *MaxwellEventBatchEncoder) EncodeDDLEvent(e *model.DDLEvent) (*MQMessage
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	var keyLenByte [8]byte
-	binary.BigEndian.PutUint64(keyLenByte[:], uint64(len(key)))
-	var valueLenByte [8]byte
-	binary.BigEndian.PutUint64(valueLenByte[:], uint64(len(value)))
 
 	keyBuf := new(bytes.Buffer)
-	var versionByte [8]byte
-	binary.BigEndian.PutUint64(versionByte[:], BatchVersion1)
-	keyBuf.Write(versionByte[:])
-	keyBuf.Write(keyLenByte[:])
 	keyBuf.Write(key)
-
 	valueBuf := new(bytes.Buffer)
-	valueBuf.Write(valueLenByte[:])
 	valueBuf.Write(value)
 	return NewMQMessage(keyBuf.Bytes(), valueBuf.Bytes(), e.CommitTs), nil
```

Comment:
```suggestion
	return NewMQMessage(key, value, e.CommitTs), nil
```
and remove L300-L304

Reply:
done